frontend/components/inbox/User.vue

```diff
@@ -102,58 +103,57 @@
 </template>
 
 <script>
-  import axios from "axios";
+import axios from "axios";
 
-  export default {
-    name: "User",
-    data() {
-      return {
-        inbox_user: null,
-        roles: [
-          {
-            "key": "GUEST",
-            "label": "Student"
-          },
-          {
-            "key": "AGENT",
-            "label": "Teaching Assistant"
-          },
-          {
-            "key": "MANAGER",
-            "label": "Coordinator"
-          }
-        ]
-      }
+export default {
+  name: "User",
+  data() {
+    return {
+      inbox_user: null,
+      roles: [
+        {
+          "key": "GUEST",
+          "label": "Student"
+        },
+        {
+          "key": "AGENT",
+          "label": "Teaching Assistant"
+        },
+        {
+          "key": "MANAGER",
+          "label": "Coordinator"
+        }
+      ]
+    }
+  },
+  mounted() {
+    const inboxId = this.$route.params.inboxId
+    const userId = this.$route.params.userId
+
+    axios.get(`/api/inboxes/${inboxId}/users/${userId}`).then(response => {
+      this.inbox_user = response.data
+    })
+  },
+  methods: {
+    onSave: function () {
+      const inboxId = this.$route.params.inboxId
+      const userId = this.$route.params.userId
+
+      axios.put(`/api/inboxes/${inboxId}/users/${userId}`, this.inbox_user).then(_ => history.back())
     },
-    mounted() {
-      axios.get(`/api${window.location.pathname}`).then(response => {
-        this.inbox_user = response.data
-      })
+    onCancel: function () {
+      window.history.back();
     },
-    methods: {
-      onSave: function () {
-        axios.defaults.xsrfCookieName = 'csrftoken';
-        axios.defaults.xsrfHeaderName = "X-CSRFTOKEN";
-
-        axios.put(`/api${window.location.pathname}`, this.inbox_user).then(_ => {
-          history.back()
-        })
-      },
-      onCancel: function () {
-        window.history.back();
-      },
-      onDelete: function () {
-        if (confirm('Are you sure?')) {
-          axios.defaults.xsrfCookieName = 'csrftoken';
-          axios.defaults.xsrfHeaderName = "X-CSRFTOKEN";
+    onDelete: function () {
+      if (confirm('Are you sure?')) {
+        const inboxId = this.$route.params.inboxId
+        const userId = this.$route.params.userId
```

Comment:
should this be a custom tailwind popup or a animation in the button? The default confirmation feels a bit unusual to me